tests/test_utils.py

```diff
@@ -377,3 +388,34 @@ def test_load_spacy_model(model_name, is_found):
     else:
         with pytest.raises(ModuleNotFoundError):
             load_spacy_model(model_name)
+
+
+@pytest.mark.parametrize(
+    "get_obj",
+    [
+        lambda: torch.tensor([8.0, 8.0, 5.0]),
+        lambda: {
+            "a": torch.tensor([8.0, 8.0, 5.0]),
+            "b": torch.tensor([3.0, 1.0, 2.0, 4.0]),
+            "c": torch.tensor([[1.0, 1.9], [1.2, 1.3]]),
+        },
+        lambda: Sequential(Linear(3, 5), Linear(5, 2)),
+    ],
+)
+def test_patched_torch_save(tmpdir, get_obj):
+    md5_sums = []
+
+    for _ in range(2):
+        torch.manual_seed(42)
+        obj = get_obj()
```

Comment:
I guess by using `lambda` you are trying to defer instantiation to this line, right? Was it necessary? I guess just become of the possibility to control random state?

Reply:
Yes, indeed, the problem was that I couldn't find another way to do that.

I wanted 2 `obj` to be created at each iteration, but with the same `seed` so that they should have equal content but different identity, `_cdata` etc.